src/testers.jl

```diff
@@ -149,8 +196,9 @@ function frule_test(f, xẋs::Tuple{Any, Any}...; rtol=1e-9, atol=1e-9, fdm=_fdm
     # TODO: add isapprox replacement that works for more types
     @test Ω_ad == Ω || isapprox(collect(Ω_ad), collect(Ω); rtol=rtol, atol=atol)
 
+    ẋs_is_dne = ẋs .== nothing
```

Comment:
Shall we be explict about what we are doing (ignoring it), vs about why we are doing it (it doesn't exist).
Unlike for `rrule` we can't check that the AD produced a `DoesNotExist` since these are inputs.
```suggestion
    ẋs_is_ignored = ẋs .== nothing
```

But maybe it is better to keep to matching the names used in `rrule_test`